github-profiles/Marcos.-0222/style.css

```diff
@@ -0,0 +1,106 @@
+@import 'https://fonts.googleapis.com/css?family=Roboto:300,400,500';
+
+header a {
+    border: 1px solid #e81c4f;
+    border-radius: 290486px;
+    color: white;
+    font-size: 0.6em;
+    letter-spacing: 0.2em;
+    padding: 1em 2em;
+    text-transform: uppercase;
+    text-decoration: none;
+    transition: none 200ms ease-out;
+    transition-property: color, background;
+  }
+  
+  header a:hover {
+    background:  #e81c4f;
+    color: white;
+  }
+body {
+  font-family: "Roboto", "Helvetica", "Arial", sans-serif;
+}
+body {
+    margin: 0 auto;
+    max-width: 50em;
+  }
+  body {
+    line-height: 1.5;
+    padding: 4em 1em;
+  }
+  
+  h2 {
+    margin-top: 1em;
+    padding-top: 1em;
+  }
+  body {
```

Comment:
You can group all your styles for your `body` in just one place, I mean:
`body { font-family: "Roboto", "Helvetica", "Arial", sans-serif;
              margin: 0 auto;
               max-width: 50em;
              line-height: 1.5;
             padding: 4em 1em;}`
You do not have to repeat the same tag every time you want to add some style to it.